hoodie-client/src/test/java/com/uber/hoodie/common/HoodieClientTestUtils.java

```diff
@@ -87,7 +87,7 @@ public static SparkConf getSparkConfForTest(String appName) {
         SparkConf sparkConf = new SparkConf()
                 .setAppName(appName)
                 .set("spark.serializer", "org.apache.spark.serializer.KryoSerializer")
-                .setMaster("local[4]");
+                .setMaster("local[1]");
```

Comment:
@vinothchandar That is true. Left a comment here last week : https://github.com/uber/hoodie/pull/235 ( sorry noticed the issue is closed, should've commented elsewhere)